slackapi/bolt-js
# Update get-manifest hook and Manifest exports

###  Summary
This PR contains some minor updates to get-manifest hook behavior and Manifest exports. Also removes some tests that were noops (😢) 

### Testing 
* Build this bolt-js branch locally, and symlink it: `npm run build:clean && npm run build && npm link`
* Make a fresh project from the TS template: `hermes create -t slack-samples/bolt-ts-starter-template -b future`
* Get your deps including the local bolt changes: `npm i && npm link @slack/bolt`
* `npm run build` on the template project to generate the manifest.js file 
* `hermes manifest` to confirm the manifest is properly generated.
* * Make a manual edit:
    * Update the export syntax in `manifest.ts`. That is what we are testing:
![Screen Shot 2022-08-02 at 9 20 45 PM](https://user-images.githubusercontent.com/55667998/182523458-35cee973-b916-4799-ad47-cb20c6e4d337.png)
    * Repeat the `npm run build && hermes manifest` to generate the manifest ![Screen Shot 2022-08-02 at 9 24 10 PM](https://user-images.githubusercontent.com/55667998/182523805-30c0cf5b-6aef-4f56-b05c-c57ee73ad80e.png)



### Requirements (place an `x` in each `[ ]`)
